tests/unit/multi-currency/test-class-multi-currency.php

```diff
@@ -380,13 +380,13 @@ public function test_update_selected_currency_by_url_updates_session_when_curren
 		$this->assertSame( 'GBP', WC()->session->get( WCPay\MultiCurrency\MultiCurrency::CURRENCY_SESSION_KEY ) );
 	}
 
-	public function test_update_selected_currency_by_geolocation_does_not_set_session_when_currency_not_enabled() {
+	public function test_update_selected_currency_by_geolocation_does_not_set_session() {
```

Comment:
I think we should keep the old name, but also add `$this->mock_localization_service->method( 'get_country_locale_data' )->with( 'CL' )->willReturn( [ 'currency_code' => 'CLP' ] );` to make it similar to other tests. Although we're already covering it while testing `update_selected_currency` directly.

Reply:
@ismaeldcom The reason of modifying this test was because now we are not setting the session for any currency, no matter if it exists or not, so I thought checking also a nonexisting currency was a bit redundant. I changed it to CA because CAD is one of the currencies added into the test. What do you think? 🤔

Reply:
I think the test is working because we're not mocking `get_country_locale_data`, if you add that line it will fail. Because we're setting the session, even after this change, [here](https://github.com/Automattic/woocommerce-payments/blob/54c638f9f3536d5520b4af6bcc4633712c468cc6/includes/multi-currency/MultiCurrency.php#L640). We're only skipping setting the cookie 🙂 

That's why I think testing the cookie existence will be better, but as I mentioned, it will be tricky to do because we rely on `headers_sent()`.